examples/controls_example/public/app/control_group_renderer_examples/edit_example.tsx

```diff
@@ -133,7 +134,8 @@ export const EditExample = () => {
               iconType="plusInCircle"
               isDisabled={controlGroupAPI === undefined}
               onClick={() => {
-                controlGroupAPI!.openAddDataControlFlyout({ controlInputTransform });
+                if (!controlGroupAPI) return;
+                controlGroupAPI!.openAddDataControlFlyout({ controlStateTransform });
```

Comment:
Is `!` needed, if statement should tell typescript that controlGroupAPI is defined

Reply:
Woops! Yup, it's no longer necessary - removed in 5c5e575c32a2454471e0b49dedefb9526cd6d05a